OpenNefia.Content/Skills/SkillsSystem.cs

```diff
@@ -92,28 +99,62 @@ private void ResetSkillBuffs(SkillsComponent skills)
             }
         }
 
+        public LevelAndPotential Ensure(SkillsComponent skills, PrototypeId<SkillPrototype> protoId)
+        {
+            if (skills.Skills.TryGetValue(protoId, out var level))
+                return level;
+
+            return new LevelAndPotential()
```

Comment:
Same as with feats above. I think it should allocate and then put the new `LevelAndPotential` in the `SkillsComponent`. Otherwise you'd just use `Level()`.

Reply:
changed in 8a38fcf